src/snowflake/snowpark/modin/pandas/utils.py

```diff
@@ -837,3 +831,65 @@ def get_as_shape_compatible_dataframe_or_series(
 
 _original_pandas_MultiIndex_from_frame = pandas.MultiIndex.from_frame
 pandas.MultiIndex.from_frame = from_modin_frame_to_mi
+
+
+def ensure_index(
+    index_like: Axes | pd.Index | pd.Series, copy: bool = False
+) -> pd.Index | pandas.MultiIndex:
+    """
+    Ensure that we have an index from some index-like object.
+
+    Parameters
+    ----------
+    index_like : sequence
+        An Index or other sequence
+    copy : bool, default False
+
+    Returns
+    -------
+    index : Index
```

Comment:
```suggestion
    Index
```

Reply:
Returned objects should not have a name specified in the docstring - this is only done for parameters.